src/Explorer/Explorer.tsx

```diff
@@ -343,28 +346,51 @@ export default class Explorer {
       return;
     }
     this._isInitializingNotebooks = true;
+    if (userContext.features.phoenix) {
+      const connectionStatus: DataModels.ContainerConnectionInfo = {
+        status: ConnectionStatusType.Allocating,
+      };
+      useNotebook.getState().setConnectionInfo(connectionStatus);
+      const provisionData = {
+        cosmosEndpoint: userContext.databaseAccount.properties.documentEndpoint,
+        resourceId: userContext.databaseAccount.id,
+        dbAccountName: userContext.databaseAccount.name,
+        aadToken: userContext.authorizationToken,
+        resourceGroup: userContext.resourceGroup,
+        subscriptionId: userContext.subscriptionId,
+      };
+      const connectionInfo = await this.phoenixClient.containerConnectionInfo(provisionData);
+      if (connectionInfo.data && connectionInfo.data.notebookServerUrl) {
+        connectionStatus.status = ConnectionStatusType.Connected;
+        useNotebook.getState().setConnectionInfo(connectionStatus);
+
+        useNotebook.getState().setNotebookServerInfo({
+          notebookServerEndpoint: userContext.features.notebookServerUrl || connectionInfo.data.notebookServerUrl,
```

Comment:
just return the data part directly from containerConnectionInfo

Reply:
Want to maintain consistency through out return types in phoenix client.... also having status in return call helps. May not be here but through out app level this helps.